product/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http              import HttpResponse, JsonResponse
+from django.views             import View
+
+from datetime                 import date, datetime
+from .models                  import Category, Product, Reward, Collection, LikeUser, ProductContent
+
+
+class CategoryView(View):
+    def get(self, request):
+        try:
+            product = Product.objects.all()
+            product_list = [{
+                'category_image'   : product.category_set.all()[0].image,
+                'title'            : product.title,
+                'goal_amout'       : product.goal_amount,
+                'toal_amount'      : product.total_amount,
+                'achieved_rate'    : product.achieved_rate,
+                'total_supporters' : product.total_supporters,
+                'closing_date'     : product.closing_date,
+                'thumbnail'        : product.thumbnail_url,
+                'category'         : product.category_set.all()[0].name,
+                'id'               : product.id,
+                'maker_info_name'  : product.maker_info.name
+                } for product in product]
+            return JsonResponse({"MESSAGE" : "SUCCESS", "data" : {'product'  : product_list}}, status=200)
```

Comment:
return 해주시는 json body 를 펼쳐보면 아래처럼 뭔가 언밸런스한 느낌을 받는데요! key 값을 소문자로 통일하고 불필요한 space 를 없애서 조금 더 깔끔하게 만들 수 있을 것 같습니다.
```json
{
    "MESSAGE" : "SUCCESS", 
    "data" : {
        'product'  : product_list
    }
}
```
```suggestion
            product = Product.objects.all()
            product_list = [{
                'category_image'   : product.category_set.all()[0].image,
                'title'            : product.title,
                'goal_amout'       : product.goal_amount,
                'toal_amount'      : product.total_amount,
                'achieved_rate'    : product.achieved_rate,
                'total_supporters' : product.total_supporters,
                'closing_date'     : product.closing_date,
                'thumbnail'        : product.thumbnail_url,
                'category'         : product.category_set.all()[0].name,
                'id'               : product.id,
                'maker_info_name'  : product.maker_info.name
                } for product in product]
            return JsonResponse({"message" : "SUCCESS", "data" : {'product' : product_list}}, status=200)
```

Reply:
수정완료 했습니다!!!